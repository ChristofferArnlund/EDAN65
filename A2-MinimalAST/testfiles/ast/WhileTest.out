Program
  List
    Func ID="a"
      List
      List
        IdDecl ID="x"
          Opt
        While
          Ge
            Name ID="x"
            Numeral NUMERAL="4"
          List
            IdDecl ID="y"
              Opt
        While
          Numeral NUMERAL="1"
          List
