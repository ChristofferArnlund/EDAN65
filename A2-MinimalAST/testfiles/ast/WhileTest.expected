Program
  List
    Func ID="a"
      List
      List
        IdDecl ID="x"
        While
          Ge
            Name ID="x"
            Numeral NUMERAL="4"
          List
            IdDecl ID="y"
        While
          Numeral NUMERAL="1"
          List
